cmd/mountlib/utils.go

```diff
@@ -89,18 +89,7 @@ func (m *MountPoint) SetVolumeName(vol string) {
 	if vol == "" {
 		vol = fs.ConfigString(m.Fs)
 	}
-	m.MountOpt.SetVolumeName(vol)
-}
-
-// SetVolumeName removes special characters from volume name if necessary
-func (o *Options) SetVolumeName(vol string) {
```

Comment:
This patch seems to have removed all the checking for the other OSes?

If we just want a valid name, then I would have thought using [encoder.OS](https://github.com/rclone/rclone/blob/master/lib/encoder/os_windows.go#L26) here and `.FromStandardName()` (not `Path`) would be enough wouldn't it?

I don't actually know what the volume names can be in macOS/FreeBSD though.